batch.py

```diff
@@ -488,11 +490,19 @@ def summarise_association_results(
     from multipy.fdr import qvalue
 
     logging.info('before glob (pv files) - summarise job')
-    existing_pv_files = list(to_path(pv_folder).glob('*_pvals.csv'))
+    storage_client = storage.Client()
+    bucket = get_config()['storage']['default']['default'].replace('gs://', '')
+    prefix = f"{get_config()['workflow']['output_prefix']}/{celltype}"
+    existing_pv_files = {
+        filepath
+        for filepath in map(
+            lambda x: f'gs://{bucket}/{x.name}',
+            storage_client.list_blobs(bucket, prefix=prefix),
```

Comment:
I think you need delimiter here to avoid filtering through all blobs in the bucket

```suggestion
            storage_client.list_blobs(bucket, prefix=prefix, delimiter='/'),
```